codalab/client/json_api_client.py

```diff
@@ -706,3 +706,10 @@ def get_workers_info(self):
         request_path = '/workers/info'
         response = self._make_request('GET', request_path)
         return response['data']
+
+    @wrap_exception('Unable to get the locations of bundles')
+    def get_multiple_bundle_locations(self, bundle_uuid):
```

Comment:
I think this name makes more sense?

```suggestion
    def get_bundle_locations(self, bundle_uuid):
```

Reply:
Three's a function named get_bundle_locations in bundle_model.py. I didn't want to cause any confusion so I named my function that way.

Reply:
I think it's fine to call this one `get_bundle_locations` -- as this method pretty much corresponds with `get_bundle_locations` in bundle_model.py, right?